<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChefFormClass</class>
 <widget class="QMainWindow" name="ChefFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChefForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Add new recipe</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Preparation time:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEditPrep"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ingredients:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditIng"/>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButtonView">
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Show only recipes for your speciality</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineEditSearch"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ChefForm.qrc"/>
 </resources>
 <connections/>
</ui>
